scripts/cms-any/dispatch-ci.php

```diff
@@ -1,13 +1,13 @@
 <?php
 
 // run on two consecutive days of the week
-$dayOfWeek = predictable_random_int(6);
+$dayOfWeek = predictable_random_int('dispatch-ci', 6);
```

Comment:
We could use `basename(__FILE__)` instead, saves having the same magic string multiple times. Change all of these if you change any.
Doesn't ultimately matter though so I won't hold up merging if you leave these as they are.

Reply:
Already tried that, it doesn't work due to the use of eval()

Reply:
Oh, yikes.... we should probably not be using eval but I guess that's something for a separate card.
I guess we just have to hope we don't forget and add code relying on `basename` in another script or during refactoring in the future.

Reply:
basname() isn't the problem, the problem is __FILE__ will be the file with eval() in it, rather than the script file

Reply:
Well, pretend I said `__FILE__` then, my remark still stands. :p
Nothing to change in this PR, just something we'll have to watch out for in the future.